### PR TITLE
Few small changes

### PR DESCRIPTION
**Pull request by TomasRoj**
I changed the working version to 1.8.9 (tested). Also changed the installation tutorial to be more simple to do.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ A collection of themes for the [Arduino](http://arduino.cc) IDE. They are compat
 #### Installation
 - Back up your current theme by rename the folder **{Arduino IDE installation folder}/lib/theme** to **theme.original** or any other convenient name.
 - Download the desired theme from the download link on the list above.
-- Copy the theme folder from the downloaded file to **{Arduino IDE installation folder}/lib**.
+- From the downloaded .zip file extract the **theme** folder to **{Arduino IDE installation folder}/lib**.
```
Comment (per1234) [inline README.md]:
```suggestion
- Unzip the downloaded file.
- Copy the **theme** folder from the unzipped folder to **{Arduino IDE installation folder}/lib**.
```

**Review by per1234 — CHANGES_REQUESTED** [review]
I agree that this step was not very clear in my original instructions. However, i think we can make it even more clear by breaking it into two steps.

**Comment by TomasRoj** [comment]
Applied your suggestion.

**Comment by per1234** [comment]
Thanks so much @TomasRoj!

**Comment by TomasRoj** [comment]
No problem! Glad to help you!

**Pull request merged**